vault/logical_system.go

```diff
@@ -930,11 +930,9 @@ func (b *SystemBackend) handlePluginRuntimeCatalogList(ctx context.Context, _ *l
 	}
 
 	resp := &logical.Response{
-		Data: map[string]interface{}{},
-	}
-
-	if len(data) > 0 {
-		resp.Data["runtimes"] = data
+		Data: map[string]interface{}{
```

Comment:
nit: with this change you can probably just return the response without assigning it to `resp`.

e.g:
```return &logical.Response..., nil```

Reply:
Yeah I think that makes the intent a little clearer, and it's always nice to eliminate an unnecessary mutable variable 👍